.taskcluster.yml

```
@@ -0,0 +1,42 @@
+version: 1
+policy:
+  pullRequests: collaborators
+tasks:
+  $let:
+    head_rev:
+      $if: tasks_for == "github-pull-request"
+      then: ${event.pull_request.head.sha}
+      else: ${event.after}
+    repository:
+      $if: tasks_for == "github-pull-request"
+      then: ${event.pull_request.head.repo.html_url}
+      else: ${event.repository.html_url}
+  in:
+    $match:
+      (tasks_for == "github-pull-request" && event["action"] in ["opened","reopened","synchronize"]):
+        taskId:
+          $eval: as_slugid("test")
+        deadline:
+          $fromNow: 1 day
```

```suggestion
          $fromNow: 1 hour
```
should be more than enough for what we're running here